sdk/src/logs/multi_log_record_processor_factory.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef ENABLE_LOGS_PREVIEW
```

Comment:
Not related to this PR, but I think we can later remove the factory class for MultiLog<Record>Processor, as this class is only used internally in SDK.